api.py

```diff
@@ -20,14 +21,62 @@ def check_authentication(handler):
 def format_member(member):
 	return {
 		'id': member.id,
+		'show': member.show,
 		'name': member.name,
 		'dce': member.dce,
-		'mailingList': member.mailing_list,
-		'currentStudent': member.current_student,
+		'mailing_list': member.mailing_list,
```

Comment:
Because this function is formatting a member to be returned as JSON, I think camelCase should be used to align with the JavaScript standard.

Reply:
You make a good point. I have made the case consistent